satellite/marketing/marketingweb/server.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/zeebo/errs"
 	"go.uber.org/zap"
 	"golang.org/x/sync/errgroup"
+	"github.com/gorilla/mux"
```

Comment:
Why do you need gorilla/mux here?

Reply:
@egonelbre  So I can use PathPrefix on line 63. That fixes the serving of static assets. Also, I have future code where I need to pass params to the endpoints which gorilla/mux handles nicely.

Reply:
The previous one http.StripPrefix and http.Handle was working.

Reply:
@egonelbre  The previous one actually wasn't working. I told @VinozzZ I would fix it in my PR since the scope of her PR was only to open the port. Also I have code I haven't put up yet where I use mux to pass params to endpoints like `/updateOffer/r{offerID}` which I like. Is that ok ? If not what do you suggest ?

Reply:
ok, sounds good